chef/cookbooks/updater/recipes/default.rb

```diff
@@ -23,6 +23,11 @@
   node.save
 
   if node[:platform_family] == "suse"
+
+    repos = Provisioner::Repositories.get_repos(node[:platform],
```

Comment:
hmm, this gets evaluated in 1st phase already.. are you sure that's safe to do? I'm unsure here.